fdbserver/DataDistribution.actor.cpp

```diff
@@ -3473,8 +3474,8 @@ ACTOR Future<Void> storageServerTracker(
 			}
 
 			//If this storage server has the wrong key-value store type, then mark it undesired so it will be replaced with a server having the correct type
-			if (hasWrongDC) {
-				TraceEvent(SevWarn, "UndesiredDC", self->distributorId)
+			if (hasWrongDCOrLocality) {
+				TraceEvent(SevWarn, "UndesiredDCOrLocality", self->distributorId)
```

Comment:
Do we want to log whether the locality is wrong in this message? Also unrelated, why do we have a detail for "WrongDC" that always logs "?"?